### PR TITLE
Backport changes from #14373 to 3.x

### PR DESCRIPTION
**Pull request by markstory**
Refs #14372

### CODE REVIEW

File: src/Http/Response.php [hunk 1]
```diff
@@ -46,6 +46,10 @@ class Response implements ResponseInterface
 {
     use MessageTrait;
 
+    public const MIN_STATUS_CODE_VALUE = 100;
```
Comment (dereuromark) [inline src/Http/Response.php]: public is not good for php5

File: src/Http/Response.php [hunk 2]
```diff
@@ -46,6 +46,10 @@ class Response implements ResponseInterface
...
+
+    public const MAX_STATUS_CODE_VALUE = 599;
```
Comment (othercorey) [inline src/Http/Response.php]:
```suggestion
    const MIN_STATUS_CODE_VALUE = 100;

    const MAX_STATUS_CODE_VALUE = 599;
```

File: src/Http/Response.php [hunk 3]
```diff
@@ -46,6 +46,10 @@ class Response implements ResponseInterface
 {
     use MessageTrait;
 
+    const MIN_STATUS_CODE_VALUE = 100;
+
+    const MAX_STATUS_CODE_VALUE = 599;
```
Comment (dereuromark) [inline src/Http/Response.php]: Usually you would group constants:

- STATUS_CODE_MAX
- STATUS_CODE_MIN

Comment (othercorey) [reply src/Http/Response.php]: Agreed. I can fix it up.

Comment (dereuromark) [reply src/Http/Response.php]: You would also have to do that in master now.

**Pull request merged**